### ISSUE TITLE
Why so many false collissions

### ISSUE THREAD

**Issue by unclevito2017** [iceland2k14/bsgs#33]
A False collision ignored. Bloom collision, but No bPfile collision.  Do I need to make the bpfile again?

**Comment by iceland2k14** [comment]
No. False collision is expected in probabilistic data type as we are using bloom filters. If it annoys you, you can just comment that message display in the code, the script is not stopping on that.
Pubsub i already updated and moved to a new Repo with both Windows and Linux [---> Here ](https://github.com/iceland2k14/quick)
